src/bridge-canister/src/operation_store.rs

```diff
@@ -25,6 +25,7 @@ where
 {
     dst_address: H160,
     payload: P,
+    memo: Option<String>,
```

Comment:
Memo size must not be unbouded as it is given by client and can be abused to drain the canister of memory. Also, Rust `String` suggests that memo should be a valid UTF-8 string, but we don't really need to enforce it.

Instead, let's just use the same memo format as ICRC-1 tokens do, `[u8; 32]`.